src/game/Strategic/Quest_Debug_System.cc

```diff
@@ -2056,9 +2056,9 @@ static void BtnQuestDebugGiveItemToNPCButtonCallback(GUI_BUTTON* btn, UINT32 rea
 {
 	if (reason & MSYS_CALLBACK_REASON_POINTER_UP)
 	{
-		OBJECTTYPE		Object;
+		OBJECTTYPE		Object = {};
```

Comment:
This isn't necessary because it is the first thing CreateItem does, but as I said, I'd prefer a proper constructor for OBJECTTYPE.